starter-code/css/main.css

```diff
@@ -6,3 +6,127 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  margin: 0;
+  border: 0;
+  font-family: sans-serif;
+}
+
+nav {
+  height: 70px;
+}
+
+.spotify-logo {
+  float: left;
+  height: 60px;
+  margin-left: 30px;
+  margin-top: 10px;
+}
+
+.section1 {
+  background-image: url(../images/landing.jpg);
+  height: 90vh;
+  background-size: cover;
+  color: white;
+  text-align: center;
+  font-size: 30px;
+}
+
+ h1 {
+  margin: 0;
+  font-size: 80px;
+  padding-top: 200px;
+}
+
+.section2 h2 {
+  font-size: 40px;
+  text-align:center;
+  text-decoration: underline solid #00B172;
+}
+
+ul {
+  list-style-type: none;
+  float: right;
+  font-size: 25px;
+}
+
+li {
+  text-decoration: none;
+  display: inline;
+  margin-right: 30px;
+}
+
+.clearfix {
+  clear: both;
+}
+
+.music-icon, .high-quality-icon, .devices-icon {
+  height: 110px;
+}
+
+.section2 {
+  display: inline;
+}
+
+.flexbox1 {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: 100px 0;
+}
+
+.element {
+  width: 400px;
+  text-align: center;
+}
+
+.element h3 {
+  color: #00B172;
+}
+
+.section3 {
+  display: flex;
+  background-color: #00B172;
+  border: 50px solid white;
+  color: white;
+  height: 90vh;
+  align-items: baseline;
+}
+
+.section3 h2 {
+  text-decoration: underline;
+  font-size: 40px;
+  margin-bottom: 70px;
+}
+
+.spotify-icon-white {
+  width: 100px;
+}
+
+.spotify-app {
+  width: 400px; 
+}
+
+.element1 {
+  align-self: flex-start;
+  margin-left: 50px;
+}
+
+.element2 {
+  align-self: center;
+  margin: 0 50px 
+}
+
+.element3 {
```

Comment:
You can play around with margin in order to display the assets as required.
But you can aswell set the element2 as a background image.
This is what I did:
```css
.container {
  background-color: #00b172;
  margin: 2vw;
  padding: 5vh;
  background-image: url(../images/spotify-icon-white.png);
  background-position: center center;
  background-size: 200px;
  background-repeat: no-repeat;
}
```